scripts/index.js

```diff
@@ -29,6 +31,10 @@
 //
 // Author:
 //   helfi92
+const got = require('got');
+
+const BUGZILLA_BASE_URL = 'https://bugzilla.mozilla.org';
```

Comment:
You can set this in the environment as well so it can be overridden. Create a `.env` file in this repo and force commit the file, but leave it in `.gitignore`.